dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtilTest.java

```diff
@@ -0,0 +1,1194 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.harvard.iq.dataverse.export.openaire;
+
+import com.google.gson.Gson;
+import edu.harvard.iq.dataverse.GlobalId;
+import edu.harvard.iq.dataverse.api.dto.DatasetDTO;
+import edu.harvard.iq.dataverse.api.dto.DatasetVersionDTO;
+import edu.harvard.iq.dataverse.api.dto.FileDTO;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ *
+ * @author francesco.cadili@4science.it
+ */
+public class OpenAireExportUtilTest {
```

Comment:
We should keep tests separate from non tests code:
`dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtilTest.java` -> `dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtilTest.java`